src/js/rating.spec.js

```diff
@@ -0,0 +1,55 @@
+const { expect } = require("chai");
+
+const Rating = require("./rating");
+// Parámetros que debe recibir: idRating, name, score, idAuthor, idRecipe
+const rating = new Rating(1, "Marco", 7, 4, 3);
+// const rating2 = new Rating(null, "Marco", 7, 4, 3);
+// const rating3 = new Rating(1, 8, 7, 4, 3);
+
+
+describe("rating.idRating", () => {
```

Comment:
Vamos a poner una limitación de un `describe` por cada archivo de test. Un `describe` puede tener muchos `it`, y también puede tener otros `describes` dentro, así que no pasa nada. Podríamos poner una estructura de este tipo, por ejemplo:

- `describe` - Rating
    - `describe` - Atributos
        - `it` - idRating debería funcionar correctamente
        - `it` - name debería funcionar correctamente
        - etc
   - `describe` - Excepciones
       - `it` - Debería lanzar una excepción si se le pasa un idRating erróneo
       - etc

Hazlo de la forma que mejor te parezca, lo importante es que solo haya un `describe` por archivo